src/components/reactions/Hate.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+import hateIcon from '../../assets/icons/thumbs-down.svg';
+
+import './Like.css';
+
+const Hate = () => {
+  const [isReacted, setIsReacted] = useState(false);
+
+  const handleClick = () => {
+    setIsReacted(!isReacted);
+  }
+
+  const clickStyle = {
+    color : isReacted ? '#000' : '#818181',
+    filter: isReacted || 'invert(56%) sepia(1%) saturate(1741%) hue-rotate(3deg) brightness(89%) contrast(86%)'
+  }
+
+  return (
+    <div className="like" type='button' onClick={handleClick}>
```

Comment:
button 태그로 표현해주면 어떨까요?
```suggestion
    <button className="like" type='button' onClick={handleClick}>
```